tests/acceptance/test_mode_transitions.py

```diff
@@ -61,7 +60,7 @@ def test_antenna_trans_to_idle(cm_manager_connected_to_antnn):
     """Test the idle function"""
     cm_manager_connected_to_antnn.set_idle_mode()
     # The antenna takes roughly 30 sec to switch modes
-    time.sleep(33)
+    time.sleep(60)
```

Comment:
In other pull requests, the antenna transition time is 30-33 sec and the tests pass. Is there something that has changed on the simulator regarding this pull request?